.eslintrc.json

```diff
@@ -10,7 +10,7 @@
   "parser": "@typescript-eslint/parser",
   "plugins": ["@typescript-eslint"],
   "parserOptions": {
-    "ecmaVersion": 9,
+    "ecmaVersion": 2021,
```

Comment:
according to the docs, this entry may not be necessary. We could drop it, or set it to `12`.

Reply:
That's true; I just changed it for consistency, even though it's not necessary.

Reply:
OK after talkign to @cds-amal more I'm just going to delete this option for simplicity.